packages/web-app/app/_ui/utils/calculatePrice.ts

```diff
@@ -0,0 +1,11 @@
+export const calculateTokenPrice = (totalContributed: number): number => {
```

Comment:
please rename the file to the same name as the function, just to avoid confusion 🙏